docs: fix mistakes and typos; improve readability with the help of AI tools (#50)
goravel/docs: security/authentication.md

3. `ExpireAt`: Expire time;
4. `IssuedAt`: Issued time;

> > If the value of err is anything other than `ErrorTokenExpired`, the payload should be nil.

Reviewer: ```suggestion
> If `err` isn't nil other than `ErrorTokenExpired`, the payload should be nil.
```
